pkgs/by-name/cu/cups-pdf-to-pdf/package.nix

```
@@ -21,7 +21,7 @@ stdenv.mkDerivation {
   buildInputs = [ cups ];
 
   postPatch = ''
-    sed -r 's|(gscall, size, ")cp |\1${coreutils}/bin/cp |' cups-pdf.c -i
+    sed -r 's|(gscall, size, ")cp |\1${lib.getExe' coreutils "cp"} |' cups-pdf.c -i
```

Would it be practical to use `substituteInPlace` and `--replace-fail` instead of `sed` here? Generally, this is preferred over using `sed`.

Something like:

```suggestion
    substituteInPlace cups-pdf.c \
      --replace-fail \
        'gscall, size, "cp ' \
        'gscall, size, "${lib.getExe' coreutils "cp"} '
```

or maybe even:

```suggestion
    substituteInPlace cups-pdf.c \
      --replace-fail \
        '"cp ' \
        '"${lib.getExe' coreutils "cp"} '
```